src/content/headerFooterContent.js

```
@@ -11,10 +11,11 @@ chrome.runtime.onMessage.addListener(function (request, sender, sendResponse) {
     logDebug("Message received:", request);
 
     switch (request.greeting) {
-        case 'unloadHeader':
-            const headerCode = document.querySelector('#header_content').value;
+        case 'unloadHeaderHTML':
+            // const headerCode = document.querySelector('#header_content').value;
+            const headerHTMLCode = document.getElementsByName('header')[0].childNodes[0];
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)